dashboard/app/controllers/pd/application_dashboard_controller.rb

```diff
@@ -1,16 +1,26 @@
 module Pd
   class ApplicationDashboardController < ApplicationController
+    include Application::RegionalPartnerTeacherconMapping
+
+    TEACHERCON = "teachercon"
+    LOCAL_SUMMER = "local summer"
```

Comment:
I wonder if we should re-use the values we already have in [shared workshop constants](https://github.com/code-dot-org/code-dot-org/blob/e46dfac58ee2eb39fa5967c426c2a25636d4731d/lib/cdo/shared_constants/pd/shared_workshop_constants.rb#L21-L23) which are available to both the server and client, rather than redefining them here and in React?

Or maybe create another set of shared constants for partners / applications. Ideally we wouldn't need to define this in so many places.